ext/random/engine_pcgoneseq128xslrr64.c

```diff
@@ -186,6 +186,10 @@ PHP_METHOD(Random_Engine_PcgOneseq128XslRr64, jump)
 		Z_PARAM_LONG(advance);
 	ZEND_PARSE_PARAMETERS_END();
 
+	if (UNEXPECTED(advance < 0)) {
+		zend_argument_value_error(1, "must be greater than or equal to 0");
+	}
```

Comment:
```
		RETURN_THROWS();
	}
```

Reply:
Please also add a test case for this

Reply:
Ah, good catch. The test should also verify that the engine state does not change when calling with an invalid value, i.e. it should call `->generate()` after the `catch()` and verify the result against an expected value for the given seed.

Reply:
Fixed